product.go

```diff
@@ -72,12 +78,99 @@ type ProductsResource struct {
 	Products []Product `json:"products"`
 }
 
+// Pagination of results
+type Pagination struct {
+	NextPageOptions		*ListOptions
```

Comment:
This likely needs to be formatted with `go fmt`. Minor issue but all go code should be formatted with go fmt.